src/setup/type-specs/sample-type-spec.json

```diff
@@ -0,0 +1,12 @@
+{
+  "types": {
```

Comment:
this file should have two types defined in it. Having two types in this one zome doesn't currently work.